docs/integrations/sources/pinterest.md

```diff
@@ -14,11 +14,10 @@ Please read [How to get your credentials](https://developers.pinterest.com/docs/
 1. [Log into your Airbyte Cloud](https://cloud.airbyte.io/workspaces) account.
 2. In the left navigation bar, click **Sources**. In the top-right corner, click **+new source**.
 3. On the Set up the source page, enter the name for the Pinterest connector and select **Pinterest** from the Source type dropdown. 
-4. Enter your `client_id`
-5. Enter your `client_secret`
-6. Enter your `refresh_token`
-7. Enter the `start_date` you want your sync to start from
-8. Click **Set up source**
+4. Enter the `start_date` you want your sync to start from
+5. Choose `OAuth2.0` in `Authorization Method` list
+6. Proceed with OAuth authentication of your account in the pop-up window 
```

Comment:
There should be a button that says: `Authenticate your account` or similar, please reflect this as a step in public docs.